WordPress/Classes/ViewRelated/Stats/Subscribers/StatsSubscribersViewModel.swift

```diff
@@ -14,12 +14,19 @@ final class StatsSubscribersViewModel {
     }
 
     func refreshData() {
+        store.updateChartSummary()
         store.updateEmailsSummary(quantity: 10, sortField: .postId)
     }
 
     // MARK: - Lifecycle
 
     func addObservers() {
+        Publishers.MergeMany(store.chartSummary)
```

Comment:
Sorry, that was my mistake, I added `mergeMany` to combine multiple observers without realizing that it requires them to be of the same type.

This is what I was looking for:

```
  Publishers.CombineLatest(
              store.chartSummary.removeDuplicates()
              store.emailsSummary.removeDuplicates(),
        )
        .sink { [weak self] _ in
            self?.updateTableViewSnapshot()
        }
        .store(in: &cancellables)
````

Reply:
Thanks for the suggestion, it works 👍 

Because `CombineLatest` sends its first value when both `chartSummary` and `emailsSummary` send a value, I thought it would prevent the chart and the email list from updating independently. 

I checked why it worked and it's due to both the publishers posting `idle` and `loading` events first, so by the time either of them returns data, the table table view is updated without waiting for the other. The sequence looks like this:

```
▿ 2 elements
  - .0 : .idle
  - .1 : .idle
▿ 2 elements
  - .0 : .loading
  - .1 : .idle
▿ 2 elements
  - .0 : .loading
  - .1 : .loading
▿ 2 elements
  ▿ .0 : State<StatsSubscribersSummaryData>
  - .1 : .loading
▿ 2 elements
  ▿ .0 : State<StatsSubscribersSummaryData>
  ▿ .1 : State<StatsEmailsSummaryData>
```

I think it's safe to use because there will always be idle/loading events, so I'll make this change 🙇